src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/JpaInductionPersistenceAdapter.kt

```diff
@@ -48,7 +48,7 @@ class JpaInductionPersistenceAdapter(
       inductionMapper.updateEntityFromDto(inductionEntity, updateInductionDto)
       inductionEntity.updateLastUpdatedAt() // force the main Induction's JPA managed fields to update
       val updatedInductionEntity = inductionRepository.saveAndFlush(inductionEntity)
-      val previousQualificationsEntity = createOrUpdatePreviousQualifications(updateInductionDto, prisonNumber)
```

Comment:
Looks like we needed the prisonNumber all along anyway; it just was never in the DTO to start with 🤷‍♂️